utility/src/main/java/org/oppia/android/util/extensions/BUILD.bazel

```diff
@@ -0,0 +1,27 @@
+"""
+General purposes utilities to manage extensions
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "bundle_extensions",
+    srcs = [
+        "BundleExtensions.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//third_party:com_google_protobuf_protobuf-lite",
+    ],
+)
+
+kt_android_library(
+    name = "live_data_ext",
+    srcs = [
+        "LiveDataExt.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
```

Comment:
Remove this visibility for now, since it doesn't seem to be used.

Reply:
Ok

Reply:
@FareesHussain PTAL now

Reply:
Removing visibility looks good, but I left another comment about removing the dependency.